Makefile.buildtests

```diff
@@ -62,7 +62,7 @@ buildtest:
 	        WERROR=$${WERROR} \
 	        LTO=$${LTO} \
 	        TOOLCHAIN=$${TOOLCHAIN} \
-	        $(MAKE) -j$(NPROC) -B 2>&1) ; \
+	        $(MAKE) -j$(NPROC) clean all 2>&1) ; \
```

Comment:
Does that make a difference? "clean" removes the whole build output directory, forcing everything to be rebuild. "-B" is totally redundant in that case. We've just added that some days ago...